crates/dc_bundle/src/definition/modifier.rs

```diff
@@ -15,3 +15,380 @@
  */
 
 include!(concat!(env!("OUT_DIR"), "/designcompose.definition.modifier.rs"));
+
+/// Implementations are copied from euclid Transform3D.
```

Comment:
Please put this in its own file, and include the license header from euclid.

Reply:
Done.